commercetools-models/src/test/java/io/sphere/sdk/carts/MultipleShippingAddressesIntegrationTest.java

```diff
@@ -0,0 +1,158 @@
+package io.sphere.sdk.carts;
+
+import com.neovisionaries.i18n.CountryCode;
+import io.sphere.sdk.carts.commands.CartUpdateCommand;
+import io.sphere.sdk.carts.commands.updateactions.*;
+import io.sphere.sdk.carts.queries.CartQuery;
+import io.sphere.sdk.carts.queries.CartQueryModel;
+import io.sphere.sdk.client.BlockingSphereClient;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.models.Address;
+import io.sphere.sdk.models.DefaultCurrencyUnits;
+import io.sphere.sdk.queries.PagedQueryResult;
+import io.sphere.sdk.test.IntegrationTest;
+import io.sphere.sdk.test.SphereTestUtils;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+
+import static io.sphere.sdk.carts.CartFixtures.withCartDraft;
+import static io.sphere.sdk.carts.CartFixtures.withFilledCart;
+import static io.sphere.sdk.products.ProductFixtures.withTaxedProduct;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MultipleShippingAddressesIntegrationTest extends IntegrationTest {
```

Comment:
I really like that you create separate integration test for each new feature 👍

Reply:
Yep it looks like a ref 👍 
Now fixed